components/common/ProfessorCard/ProfessorCard.tsx

```diff
@@ -1,3 +1,7 @@
+// Only problem is there's a black sillhoute that appears
+// Because of the smaller professor card. Most likely due to the css element.
```

Comment:
This should be resolved before merging if possible.

Reply:
I think this ought to be fixed when we put all the components together as this isn't really a problem with the professor card but just a side effect of displaying it in the current UI.